lib/analysis/network.py

```diff
@@ -468,13 +467,26 @@ def create_message_time_graph(log_dict, nicks, nick_same_list):
     G = util.to_graph(nick_same_list)
     conn_comp_list = list(connected_components(G))
 
+    def build_graphs(nick_sender,nick_receiver,line):
```

Comment:
would it be better if we shift build_graphs and compare_nick to util? What do you think? They can be used in other modules as well.
moreover, as a novice user i will not know what exactly you mean by 'line' in the argument. change that?